egs/iwslt21/asr1/run.sh

```diff
@@ -126,7 +126,7 @@ if [ ${stage} -le 0 ] && [ ${stop_stage} -ge 0 ]; then
     # additionally we copy text to text.${case}
     for x in tr_${data_code} dt_${data_code} et_${data_code}_dev et_${data_code}_test; do
         for case in tc lc lc.rm; do
-            cp data/${x}/text data/${x}/text.${case}
+            paste -d " " <(cut -d " " -f 1 data/${x}/text) <(cut -d " " -f 2- data/${x}/text | lowercase.perl | tokenizer.perl -l en -q) > data/${x}/text.${case}
```

Comment:
it seems that we also do the `lowercase` for case `tc`, is it intentional?

Reply:
I understand it's for asr training, so this might not be that important though.

Reply:
@ftshijt Because TEDLIUM2 is already lowercased, I think it's fine.

Reply:
Cool~